src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -282,9 +282,9 @@ public static TransformationStatus DecodeInPlace(Span<byte> buffer, out int byte
             return TransformationStatus.InvalidData;
         }
 
-        class FromBase64 : ITransformation
+        class FromBase64 : Transformation
```

Comment:
These classes should be sealed. Based on conversations with @russellhadley, this will give JIT a better chance at de-virtualizing the calls.